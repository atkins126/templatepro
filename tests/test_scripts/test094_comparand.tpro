{{if intvalue0|eq,0}}
10.intvalue0 = 0
{{endif}}
---
{{if intvalue0|ge,0}}
20.intvalue0 = 0
{{endif}}
---
{{if intvalue0|lt,1}}
30.intvalue0 = 0
{{endif}}
---
{{if !intvalue0|lt,-1}}
40.intvalue0 = 0
{{endif}}
---
{{if intvalue0|le,0}}
50.intvalue0 = 0
{{endif}}
---
REF DATE: {{:valuedate|datetimetostr}}
---
{{if valuedate|eq,"2024-08-20"}}
60.valuedate = "2024-08-20"
{{endif}}
---
{{if !valuedate|ne,"2024-08-20"}}
61.valuedate = "2024-08-20"
{{endif}}
---
{{if valuedate|ge,"2024-08-20"}}
70.valuedate = "2024-08-20"
{{endif}}
---
{{if valuedate|ge,"2024-08-19"}}
71.valuedate = "2024-08-20"
{{endif}}
---
{{if !valuedate|ge,"2025-08-20"}}
72.valuedate = "2024-08-20"
{{endif}}
---
{{if valuedate|le,"2024-08-20"}}
80.valuedate = "2024-08-20"
{{endif}}
---
{{if valuedate|le,"2024-08-21"}}
81.valuedate = "2024-08-20"
{{endif}}
---
{{if valuedate|le,"2025-08-20"}}
82.valuedate = "2024-08-20"
{{endif}}
