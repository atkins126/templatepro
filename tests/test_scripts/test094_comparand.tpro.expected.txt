10.intvalue0 = 0
---
20.intvalue0 = 0
---
30.intvalue0 = 0
---
40.intvalue0 = 0
---
50.intvalue0 = 0
---
REF DATE: 2024-08-20
---
60.valuedate = "2024-08-20"
---
61.valuedate = "2024-08-20"
---
70.valuedate = "2024-08-20"
---
71.valuedate = "2024-08-20"
---
72.valuedate = "2024-08-20"
---
80.valuedate = "2024-08-20"
---
81.valuedate = "2024-08-20"
---
82.valuedate = "2024-08-20"
